open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/ViewServerConfigurationClient.java

```diff
@@ -175,24 +175,30 @@ public void setViewServicesConfiguration(List<ViewServiceConfig> viewServices) t
     /**
      * Enable a single view service.
      *
+     * @param partnerOMASServerURLRoot URL root of the OMAG Server Platform where the access service used by this view service is running
+     * @param partnerOMASServerName name of metadata access server where the access service used by this view service is running
      * @param serviceURLMarker string indicating which view service it is configuring
      * @param viewServiceOptions property name/value pairs used to configure the view service
      *
      * @throws OMAGNotAuthorizedException the supplied userId is not authorized to issue this command.
      * @throws OMAGInvalidParameterException invalid parameter.
      * @throws OMAGConfigurationErrorException unusual state in the admin server.
      */
-    public void configureViewService(String              serviceURLMarker,
```

Comment:
The comments say "enable" but the method names contains configure. I suggest aligning these names.